README.md

```diff
@@ -30,7 +30,7 @@ The following section is useful for users of Dockstore (e.g. those that want to
 launch tools). 
 
 After registering at [dockstore.org](https://dockstore.org), you will be able to download the Dockstore 
-CLI at https://dockstore.org/onboarding
+CLI at https://dockstore.org/onboarding The dockstore command line script should be installed in a location in your path.
```

Comment:
Missing period between `https://dockstore.org/onboarding` and `The dockstore command line script should be installed in a location in your path`